package/src/custom/modules/Leads/ApplicantProgrammingScore.php

```diff
@@ -15,6 +21,12 @@ class ApplicantProgrammingScore
      */
     public function updateProgrammingScore($bean, $event, $arguments)
     {
+        //Set the security context to better track the source of changes
+        $context = Container::getInstance()->get(Context::class);
+        $subject = new ApsSubject($bean->id);
+        $context->activateSubject($subject);
```

Comment:
Sergey had a suggestion of using a try catch finally block to make sure subject is deactivated when exceptions are thrown.